.github/workflows/ci-a11y.yml

```diff
@@ -0,0 +1,63 @@
+name: Accessibility tests
```

Comment:
follow the naming convention of other CI workflows and prefix with "CI" and use "Title Case"

<img width="326" alt="image" src="https://github.com/navapbc/template-application-nextjs/assets/447859/64c9bb9a-723a-406b-a04d-ad206169d67f">